core/src/test/java/tech/tablesaw/api/DoubleColumnTest.java

```diff
@@ -30,6 +29,14 @@ public void unique() {
     assertArrayEquals(arr, new double[] {1.0, 2.0, 3.0, 4.0, 5.0});
   }
 
+  @Test
+  public void createThenSortAscending() {
```

Comment:
How is this different than the existing `sortAscending` test?

Reply:
When you create an empty column like DoubleColumn.create("test") it actually creates a `DEFAULT_ARRAY_SIZE` array which has zeros. It showcases the bug nicely.